.github/workflows/slither.yml

```diff
@@ -0,0 +1,16 @@
+name: Slither Analysis
+on: [push]
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Run Slither
+        uses: crytic/slither-action@v0.2.0
+        id: slither
+        with:
+          node-version: 16
+          fail-on: high
```

Comment:
@andrew-blox Fail when high severity issues found. Although all the issues should be checked (medium and low), there could be some false positives that prevent the task from being completed. Tweaking the code to skip them will add unnecessary noise.

Reply:
as we talked maybe down the road a better solution will be to put on low and exclude whats needed in the slither-args instead